pkg/loader/compose/compose.go

```diff
@@ -398,3 +402,10 @@ func handleServiceType(ServiceType string) (string, error) {
 func normalizeServiceNames(svcName string) string {
 	return strings.Replace(svcName, "_", "-", -1)
 }
+
+func checkLabelsPorts(noOfPort int, labels string, svcName string) error {
+	if noOfPort == 0 && labels == "NodePort" || labels == "LoadBalancer" {
+		return errors.Errorf("%s defined in service %s with no ports present. Issues may occur when bringing up artifacts.", labels, svcName)
```

Comment:
Let's change message a bit. It is not possible to create Headless service with type NodePort or LoadBalancer.
The Message should be more definite.
For example something in those lines:
"kompose.service.type can't be set if service doesn't expose any ports."